app/src/main/java/org/mozilla/fenix/HomeActivity.kt

```diff
@@ -350,8 +351,16 @@ open class HomeActivity : LocaleAwareAppCompatActivity(), NavHostActivity {
         if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU &&
             !NotificationManagerCompat.from(applicationContext).areNotificationsEnabledSafe()
         ) {
+            if (settings().numberOfAppLaunches > 1) {
```

Comment:
❓ should we combine this with the condition above?

Reply:
Yes, I think you're right.  I'll update

Reply:
Coolio! Looks good!